pyo3-ffi/src/cpython/frameobject.rs

```diff
@@ -20,55 +21,10 @@ pub struct PyTryBlock {
     pub b_level: c_int,
 }
 
-#[repr(C)]
-#[cfg(not(any(PyPy, GraalPy, Py_3_11)))]
-pub struct PyFrameObject {
```

Comment:
We should add a `5154.removed.md` to note that we no longer offer access to the frame object internal structure on versions before 3.11.